cli/cli.go

```diff
@@ -27,6 +27,7 @@ type Flags struct {
 	ResultFile              string
 	ChrootPath              string
 	Database                string
```

Comment:
Let's rename this to something like MySQLDatabase now that it's specific to MySQL.